### PR TITLE
Fix typos in .tridactylrc

### PR DESCRIPTION
**Pull request by xeruf**
XDG_CONFIG and fixamo documentation

### CODE REVIEW

File: .tridactylrc [hunk 1]
```diff
@@ -122,23 +122,23 @@
...
 " fixamo_quiet
 " 
-" " Equivalent to `set csp clobber` before it was removed. This weakens your
-" " defences against cross-site-scripting attacks and other types of
-" " code-injection by reducing the strictness of Content Security Policy on
-" " every site in a couple of ways.
+" " Equivalent to `set csp clobber` before it was removed.
+" " This weakens your defences against cross-site-scripting attacks
+" " and other types of code-injection by reducing the strictness 
+" " of Content Security Policy onevery site in a couple of ways.
```
Comment (bovine3dom) [inline .tridactylrc:128]: I'm not sure what you wanted to change here but onevery isn't a word :)

Maybe "on all sites" would be clearer?

Comment (xeruf) [reply .tridactylrc]: thanks, fixed :) does this paragraph and the following ones relate to the line below or `fixamo_quiet`?
This is not quite clear to me here.

Comment (bovine3dom) [reply .tridactylrc]: It relates to the line below - the docs are directly above the commands

Comment (xeruf) [reply .tridactylrc]: Alright, added a clarification

**Comment by xeruf** [comment]
Ready :)

**Comment by bovine3dom** [comment]
Thanks!

**Pull request merged**